adr/query_pushdown.md

```diff
@@ -0,0 +1,42 @@
+Select queries of the form `{_id: <ObjectID>}` (value type is `ObjectID`).
+
+## Postgres
+
+If value type is not `ObjectID`, fallback to fetch entire table.
```

Comment:
What do you mean by `ObjectID`? A specific type, a set of types? I don't see `ObjectID` mentioned in the article linked above.